scout/server/blueprints/variant/templates/variant/sv-variant.html

```diff
@@ -67,8 +67,18 @@
          {{variant.hgnc_symbols|sort|join(", ")}}
        </div>
     </div>
-
   {% endif %}
+
+   <div class="row">
+     <div class="col-12">
+       {% if variant.category == "cancer_sv" %}
+        {{matching_variants(managed_variant, causatives, variant.matching_tiered) }}
+       {% else %}
+        {{matching_variants(managed_variant, causatives) }}
```

Comment:
Why invoking twice this macro?

Reply:
Where is the other invokation? Or you mean like why different for cancer and RD?
If the latter, I'm not 100% how unpopulated the matching_tired would be for an RD case. Do you know?

Reply:
matching_variants is a macro from components.html. You call it twice (line 75 and 77)

Reply:
> I'm not 100% how unpopulated the matching_tired would be for an RD case. Do you know?

Nope!

Reply:
Ah no, sorry, I misread! Still half asleep! 😆

Reply:
Hihi, mornings! Best time for some coffee!